v3/spring-cloud-starter-aws-v3-secrets-manager-config/src/test/java/io/awspring/cloud/v3/autoconfigure/secretsmanager/AwsSecretsManagerConfigDataLocationResolverTest.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2013-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.awspring.cloud.v3.autoconfigure.secretsmanager;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.apache.commons.logging.LogFactory;
+import org.junit.jupiter.api.Test;
+
+import org.springframework.boot.BootstrapRegistry;
+import org.springframework.boot.context.config.ConfigDataLocation;
+import org.springframework.boot.context.config.ConfigDataLocationResolverContext;
+import org.springframework.boot.context.config.Profiles;
+import org.springframework.boot.context.properties.bind.Binder;
+import org.springframework.mock.env.MockEnvironment;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+/**
+ * Tests for {@link AwsSecretsManagerConfigDataLocationResolver}
+ */
+class AwsSecretsManagerConfigDataLocationResolverTest {
```

Comment:
starters must not contain any class just the pom file with the dependencies to `autoconfigure` and `spring-cloud-aws-v3-secrets-manager` modules

Reply:
Agree. That would be a better approach. We should do the same with parameter store as well.